listenbrainz/domain/spotify.py

```diff
@@ -95,7 +95,7 @@ def get_spotify_oauth():
     """
     client_id = current_app.config['SPOTIFY_CLIENT_ID']
     client_secret = current_app.config['SPOTIFY_CLIENT_SECRET']
-    scope = 'user-read-recently-played'
+    scope = 'user-read-recently-played user-read-currently-playing'
```

Comment:
Should this be in a config file?

Reply:
I'm ok with it being in the code. Because you need to change more code if you change these values, no?